### PR TITLE
Update messages.oc.yml

### PR DESCRIPTION
**Pull request by JeanGauthier**
Some better wording and some translations of new strings

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ config:
         user_info: 'Mon compte'
         password: 'Senhal'
         rules: "Règlas d'etiquetas automaticas"
-        # ignore_origin: 'Ignore origin rules'
+        ignore_origin: 'Règlas d’ignorar l’origina'
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml:63]: You need to add a backslash before the `'`
Or do like that: 

```
ignore_origin: "Règlas d’ignorar l’origina"
```

Comment (tcitworld) [reply src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml:63]: Nope, `'` and `’` the same quote, so it's fine.

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 2]
```diff
@@ -180,25 +180,25 @@ config:
...
         #     ignore_origin_rules_definition_description: 'They are used by wallabag to automatically ignore an origin address after a redirect.<br />If a redirect occurs while fetching a new entry, all the ignore origin rules (<i>user defined and instance defined</i>) will be used to ignore the origin address.'
-        #     how_to_use_them_title: 'How do I use them?'
+            how_to_use_them_title: 'Cossí los utilizi ?'
         #     how_to_use_them_description: 'Let us assume you want to ignore the origin of an entry coming from « <i>rss.example.com</i> » (<i>knowing that after a redirect, the actual address is example.com</i>).<br />In that case, you should put « host = "rss.example.com" » in the <i>Rule</i> field.'
         #     variables_available_title: 'Which variables and operators can I use to write rules?'
         #     variables_available_description: 'The following variables and operators can be used to create ignore origin rules:'
         #     meaning: 'Meaning'
         #     variable_description:
-        #         label: 'Variable'
+                label: 'Variabla'
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml:193]: Seems to be where the CI fails

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml (hunk 1 above)
Comment (JeanGauthier) [reply src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml:63]: Absolutely, I am more into the curly one :)

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml (hunk 2 above)
Comment (JeanGauthier) [reply src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml]: changed it but still not okay =/

File: src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml [hunk 3]
```diff
@@ -375,30 +375,30 @@ howto:
...
+        open_original: "Dobrir l'URL originala de l'article"
+        toggle_favorite: "Cambiar l'estatut Favorit per l'article"
+        toggle_archive: "Cambiar l'estatut Legit per l'article"
         delete: Suprimir l'article
         material_title: Acorchis solament disponibles amb lo tèma Material
         add_link: Apondre un acorchi
         hide_form: Rescondre lo formulari actual (recèrca o nòu ligam)
         arrows_navigation: Navigar dins los articles
-        open_article: Afichar l'article seleccionat
+        open_article: "Afichar l'article seleccionat"
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml:391]: Double quote aren’t necessary, you can remove them

Comment (tcitworld) [reply src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml:391]: Won't the quote cause issues?

Comment (j0k3r) [reply src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml:391]: Oh maybe

**Pull request merged**